### PR TITLE
node-red-gpio: update patchsets

### PR DESCRIPTION
**Pull request by fr0st61te**
Bump and update patchsets with updated gpio-din node.

Signed-off-by: Ivan Mikhaylov <ivan.mikhaylov@siemens.com>

### CODE REVIEW

**Comment by fr0st61te** [comment]
Looks like it solves the problem with updating from palette manager (cause it bumps right into 0.1.0) https://support.industry.siemens.com/forum/ww/en/posts/problems-with-node-red-intel-gpio-after-updating-nodes/285581/?page=0&pageSize=10 .

File: recipes-app/node-red-gpio/node-red-gpio_0.0.6-IOT2050.bb [hunk 1]
```diff
@@ -17,12 +17,9 @@ MAINTAINER = "chao.zeng@siemens.com"
 SRC_URI = " \
     git://github.com/node-red/node-red-nodes;protocol=https \
     file://0001-add-the-board-info-add-the-led-control-node.patch \
-    file://0002-extend-gpio-to-D19.patch \
-    file://0003-Clean-up-mraa-objects-on-node-closing.patch \
-    file://0004-Add-USER-button.patch \
-    file://0005-Add-configurable-GPIO-mode-for-DIN.patch \
-    file://0006-node-red-gpio-Add-support-for-initial-message-from.patch"
-SRCREV="3087e8e2a1ea189f394bca0a2af159ad859d7722"
+    file://0002-Clean-up-mraa-objects-on-node-closing.patch \
```
Comment (jan-kiszka) [inline recipes-app/node-red-gpio/node-red-gpio_0.0.6-IOT2050.bb:20]: I think we can submit this change upstream as well. The interface was added to mraa, wasn't it?

File: recipes-app/node-red-gpio/node-red-gpio_0.0.6-IOT2050.bb [hunk 2]
```diff
@@ -17,12 +17,9 @@ MAINTAINER = "chao.zeng@siemens.com"
 SRC_URI = " \
     git://github.com/node-red/node-red-nodes;protocol=https \
     file://0001-add-the-board-info-add-the-led-control-node.patch \
```
Comment (jan-kiszka) [inline recipes-app/node-red-gpio/node-red-gpio_0.0.6-IOT2050.bb:19]: Why was that not submitted to upstream?

File: recipes-app/node-red-gpio/node-red-gpio_0.0.6-IOT2050.bb [hunk 3]
```diff
@@ -17,12 +17,9 @@ MAINTAINER = "chao.zeng@siemens.com"
...
-    file://0002-extend-gpio-to-D19.patch \
-    file://0003-Clean-up-mraa-objects-on-node-closing.patch \
-    file://0004-Add-USER-button.patch \
-    file://0005-Add-configurable-GPIO-mode-for-DIN.patch \
-    file://0006-node-red-gpio-Add-support-for-initial-message-from.patch"
-SRCREV="3087e8e2a1ea189f394bca0a2af159ad859d7722"
+    file://0002-Clean-up-mraa-objects-on-node-closing.patch \
+    file://0003-Add-USER-button.patch"
```
Comment (jan-kiszka) [inline recipes-app/node-red-gpio/node-red-gpio_0.0.6-IOT2050.bb:21]: Also here: Not yet ready for upstream?

**Review by jan-kiszka — APPROVED** [review]
Just left comments on the status of the remaining downstream patches. The changes themselves look good.

File: recipes-app/node-red-gpio/node-red-gpio_0.0.6-IOT2050.bb (hunk 1 above)
Comment (fr0st61te) [reply recipes-app/node-red-gpio/node-red-gpio_0.0.6-IOT2050.bb:20]: Yes, we can, I'm planning to do it, just waiting for mraa to be pushed.

File: recipes-app/node-red-gpio/node-red-gpio_0.0.6-IOT2050.bb (hunk 2 above)
Comment (fr0st61te) [reply recipes-app/node-red-gpio/node-red-gpio_0.0.6-IOT2050.bb:19]: I've submitted only related to gpio din - pwm, led and some fixes I'll put in different pr after mraa gets resolved to or I may put it in node-red-nodes on next week, all this about just to make everything in sync and step by step.

File: recipes-app/node-red-gpio/node-red-gpio_0.0.6-IOT2050.bb (hunk 3 above)
Comment (fr0st61te) [reply recipes-app/node-red-gpio/node-red-gpio_0.0.6-IOT2050.bb:21]: ready, just waiting for mraa but I can push in node-red-nodes until mraa is on review process...

**Pull request closed without merging**